src/shared/dm-util.c

```diff
@@ -5,3 +5,43 @@
 #include "dm-util.h"
 #include "fd-util.h"
 #include "string-util.h"
+
+int dm_deferred_remove_cancel(const char *name) {
+        _cleanup_close_ int fd = -1;
+        size_t dm_size = sizeof(struct dm_ioctl) + sizeof(struct dm_target_msg) + STRLEN("@cancel_deferred_remove") + 1;
+        struct {
+                struct dm_ioctl dm_ioctl;
+                struct dm_target_msg dm_target_msg;
+                char msg_text[STRLEN("@cancel_deferred_remove") + 1];
+        } message = {
```

Comment:
hmm, so the DM structures appear to be aligned manually already, so that packing them isn't strictly necessary. However, you might still want to add a `_packed_` before the `struct` so that we are sure no padding is inserted by the compiler anyway in our encapsulating structure.

Reply:
sounds good, added